config/hof-defaults.js

```diff
@@ -30,6 +30,7 @@ const defaults = {
       return convertPage(page);
     }
   },
+  deIndexForm: process.env.DEINDEX_FORM || 'false',
```

Comment:
is this needed for the work on postcode lookup?

Reply:
Accidentally left in from the de-indexing changes. Removed this now.